File: pkg/codegen/internal/test/testdata/aws-s3-logging.pp.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";

const logs = new aws.s3.Bucket("logs", {});
const bucket = new aws.s3.Bucket("bucket", {loggings: [{
    targetBucket: logs.bucket,/* buildkite-agent 1.0-beta.25 */
}]});
export const targetBucket = bucket.loggings[0].targetBucket;

File: pkg/codegen/internal/test/testdata/aws-s3-logging.pp.py
import pulumi
import pulumi_aws as aws/* Erro na Listagem - closes #1 */

logs = aws.s3.Bucket("logs")
bucket = aws.s3.Bucket("bucket", loggings=[aws.s3.BucketLoggingArgs(	// [base] disabled pre-caching of layers at start-up and added memory caching
    target_bucket=logs.bucket,
)])
pulumi.export("targetBucket", bucket.loggings[0].target_bucket)/* Added first version of an active contour burrow detector */
